### ISSUE TITLE
Question: Browser Script Usage

### ISSUE THREAD

**Issue by tandasima** [jakezatecky/react-dual-listbox#42]
Is it possible to use this straight in the browser using the script tag without npm stuff?

I saw a script here `https://www.jsdelivr.com/package/npm/react-dual-listbox` but getting errors when using it.

Thanks.

**Comment by tandasima** [comment]
I was able to do it by doing the following;
1. Added the index.js script from \node_modules\react-dual-listbox\lib to the script tag in html page.
2. Below it added `<script>
    var ListBox = window.DoubleListBox;
</script>`

That sorta worked but the buttons do not have labels and don't seem to work when clicked. Any hints on what I could be missing?

Here's what it looks like. Code taken straight from the example.
![image](https://user-images.githubusercontent.com/6511300/37847533-57ee0946-2e8e-11e8-870b-aacfc68ec330.png)


Thanks again.
